pkg/innerring/innerring.go

```diff
@@ -574,6 +576,8 @@ func New(ctx context.Context, log *zap.Logger, cfg *viper.Viper, errChan chan<-
 		if err != nil {
 			return nil, fmt.Errorf("init Sidechain witness scope: %w", err)
 		}
+
+		server.log.Info("autodeploy completed")
```

Comment:
this behavior change isnt related to the refactor topic so deserves separate commit